sys/androidmedia/gstamcvideodechybris.c

```diff
@@ -1820,7 +1823,9 @@ gst_amc_video_dec_handle_frame (GstVideoDecoder * decoder,
   /* 1 sec is enough for the first video input buffer from the hybris layer to
```

Comment:
This comment needs updating as the wait_buff_us is no longer 1 second, but 1/10th of a second.